doc/developer/client_side.rst

```diff
@@ -0,0 +1,90 @@
+.. _developer_client_side:
+
+Client-side development
+=======================
+
+CGXP
+----
+
+The UI of c2cgeoportal applications is built from components of the CGXP
+JavaScript library. This library is on GitHub:
+https://github.com/camptocamp/cgxp.
```

Comment:
No CGXP in master exempt in the Legacy chapter

Reply:
I have not looked yet at the developer guide. This PR is only about the integrator guide... I would prefer to do one step at a time.